indra/newview/llvoavatar.cpp

```diff
@@ -243,7 +243,8 @@ struct LLTextureMaskData
 
 struct LLAppearanceMessageContents: public LLRefCount
 {
-    LLAppearanceMessageContents():
+    LLAppearanceMessageContents(U8 num_tes):
+        mTEContents(num_tes),
```

Comment:
`LLTEContents `now requires "num_textures" argument in its ctor, so `LLAppearanceMessage `(which has a `LLTEContents `data member) also requires the same argument so it can pass it over.